public/js/components/Utilities/Modal.spec.js

```diff
@@ -5,15 +5,15 @@ import {shallow} from 'enzyme';
 
 test('Should render null when closed', () => {
   const component = renderer.create(
-    <Modal isOpen={false}><div>Modal Contents</div></Modal>
+    <Modal isOpen={false} dismiss={() => {}}><div>Modal Contents</div></Modal>
```

Comment:
I like that this is explicitly an empty function, but should we make a new jest function and pass that in?